File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		int result = num;
		int cycle = 0;
		
		while(true){
			int a = result/10, b = result%10;
			result = b * 10 + (a + b)%10;
			cycle++;
			if(result == num)
				break;
		}
		System.out.println(cycle);
		sc.close();
	}
}

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
n = int(input())
cycle = 0
a = b = 0
result = n
while(True):
    cycle += 1
    a = result // 10
    b = result % 10
    result = b * 10 + (a + b) % 10
    if(result == n):
        break
print(cycle)
